lib/Backend/Inline.cpp

```diff
@@ -4927,7 +5028,16 @@ Inline::MapFormals(Func *inlinee,
                 }
                 break;
             }
-
+        case Js::OpCode::LdAsmJsEnv:
+            if (instr->m_func == inlinee)
+            {
+                instr->SetSrc1(funcObjOpnd);
```

Comment:
Can this never be an AddrOpnd in asmjs?

Reply:
nope, it's always a RegOpnd

---
In reply to: [138230447](https://github.com/Microsoft/ChakraCore/pull/3681#discussion_r138230447) [](ancestors = 138230447)